src/sentry/relay/config/__init__.py

```diff
@@ -351,6 +351,10 @@ def get_project_config(
 
 
 def get_dynamic_sampling_config(timeout: TimeChecker, project: Project) -> Mapping[str, Any] | None:
+    if options.get("dynamic-sampling.config.killswitch"):
+        # This killswitch will cause extra load, and should only be used for AM1->AM2 migration.
+        return None
```

Comment:
@wmak you double checks this ships without this tripping, yeah? Just want to make sure

Reply:
🤔 Yeah, I think if you're concerned about this change we could take the extra step to return a manually set config from options. But that might not be worth the extra complexity.

Reply:
Yeah I still don't know that a custom config as a signal will work since we'd have to confirm it doesn't break relay in some way.

Reply:
:+1: Makes sense to me